config/locales/en.yml

```
@@ -473,6 +476,7 @@ en:
       content_policies:
         comment: Internal note
         description_html: You can define content policies that will be applied to all accounts from this domain and any of its subdomains.
+        limited_federation_mode_description_html: You are chose whether to allow federation with this domain.
```

I'm sorry I rewrote this mid-sentence, I didn't notice I ended up with a broken sentence.

```suggestion
        limited_federation_mode_description_html: You can chose whether to allow federation with this domain.
```